### PR TITLE
Add color, Sprint is coming!

### PR DESCRIPTION
**Pull request by alorma**
This is how it will look with this PR:

photo_2015-06-18_19-36-03

Used: Android support palette

[Review on Reviewable](https://reviewable.io/reviews/andrew-quebe/planets-gradle/6)

### CODE REVIEW

**Comment by Andrew-Quebe** [comment]
Are the colors dynamically changed based on the picture?

**Comment by Andrew-Quebe** [comment]
By the way, this looks amazing!

**Comment by alorma** [comment]
Yes, colors come from the picture

https://www.bignerdranch.com/blog/extracting-colors-to-a-palette-with-android-lollipop/

File: App/src/main/java/com/andrewq/planets/planets/BasePlanetActivity.java [hunk 1]
```diff
@@ -0,0 +1,351 @@
+package com.andrewq.planets.planets;
```
Comment (Andrew-Quebe) [inline App/src/main/java/com/andrewq/planets/planets/BasePlanetActivity.java]: Another question: what's this for?

Comment (alorma) [inline App/src/main/java/com/andrewq/planets/planets/BasePlanetActivity.java]: Oh sorry.

Was a try to make a base activity for each planet, and i forget to not add it into git.

I removed

**Comment by Andrew-Quebe** [comment]
No problem. I really like your work. If you can, add your info into the contributors activity so you'll be credited in the app.

**Comment by Andrew-Quebe** [comment]
If you do that before I merge, that'd be great.

**Comment by Andrew-Quebe** [comment]
Thank you for your work!

**Comment by Andrew-Quebe** [comment]
Umm...problem:

This is happening when I launch the app...
```
06-18 18:32:34.986    3391-3391/com.andrewq.planets E/AndroidRuntime﹕ FATAL EXCEPTION: main
    Process: com.andrewq.planets, PID: 3391
    java.lang.OutOfMemoryError: Failed to allocate a 3145740 byte allocation with 2021820 free bytes and 1974KB until OOM
            at dalvik.system.VMRuntime.newNonMovableArray(Native Method)
            at android.graphics.BitmapFactory.nativeDecodeAsset(Native Method)
            at android.graphics.BitmapFactory.decodeStream(BitmapFactory.java:609)
            at android.graphics.BitmapFactory.decodeResourceStream(BitmapFactory.java:444)
            at android.graphics.BitmapFactory.decodeResource(BitmapFactory.java:467)
            at android.graphics.BitmapFactory.decodeResource(BitmapFactory.java:497)
            at com.andrewq.planets.fragments.FragmentPlanets$MyAdapter.getView(FragmentPlanets.java:1196)
            at com.nhaarman.listviewanimations.BaseAdapterDecorator.getView(BaseAdapterDecorator.java:84)
            at com.nhaarman.listviewanimations.swinginadapters.AnimationAdapter.getView(AnimationAdapter.java:133)
            at android.widget.AbsListView.obtainView(AbsListView.java:2344)
            at android.widget.GridView.onMeasure(GridView.java:1060)
            at android.view.View.measure(View.java:17430)
            at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:727)
            at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:463)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.FrameLayout.onMeasure(FrameLayout.java:430)
            at android.view.View.measure(View.java:17430)
            at android.widget.RelativeLayout.measureChild(RelativeLayout.java:697)
            at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:481)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.FrameLayout.onMeasure(FrameLayout.java:430)
            at android.view.View.measure(View.java:17430)
            at android.support.v4.widget.DrawerLayout.onMeasure(DrawerLayout.java:868)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.FrameLayout.onMeasure(FrameLayout.java:430)
            at android.support.v7.internal.widget.ContentFrameLayout.onMeasure(ContentFrameLayout.java:124)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.FrameLayout.onMeasure(FrameLayout.java:430)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.FrameLayout.onMeasure(FrameLayout.java:430)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1436)
            at android.widget.LinearLayout.measureVertical(LinearLayout.java:722)
            at android.widget.LinearLayout.onMeasure(LinearLayout.java:613)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:5463)
            at android.widget.FrameLayout.onMeasure(FrameLayout.java:430)
            at com.android.internal.policy.impl.PhoneWindow$DecorView.onMeasure(PhoneWindow.java:2560)
            at android.view.View.measure(View.java:17430)
            at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:2001)
            at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1166)
            at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1372)
            at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1054)
            at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:5779)
            at android.view.Choreographer$CallbackRecord.run(Choreographer.java:767)
            at android.view.Choreographer.doCallbacks(Choreographer.java:580)
            at android.view.Choreographer.doFrame(Choreographer.java:550)
            at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:753)
            at android.os.Handler.handleCallback(Handler.java:739)
            at android.os.Handler.dispatchMessage(Handler.java:95)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5221
```

**Pull request merged**